src/main/cpp/audiobuffer.cpp

```diff
@@ -42,6 +42,7 @@ AudioBuffer::~AudioBuffer()
         delete[] _buffers->back(), _buffers->pop_back();
     }
     delete _buffers;
+    _buffers = nullptr;
 }
```

Comment:
Some files omitted the explicit assignment of deleted members to _nullptr_ in their destructors. While technically this wasn't an issue in the engine, it is consistent with the memory deallocation used elsewhere in the engine.